sdk/recoveryservices-datareplication/Azure.ResourceManager.RecoveryServicesDataReplication/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("RecoveryServicesDataReplication")]
```

Comment:
According to the swagger pr, the value here is `Microsoft.DataReplication`.

Reply:
Our approved RP namespace is "Microsoft.RecoveryServicesDataReplication" from https://github.com/Azure/azure-sdk-pr/issues/1220. I will update swagger pr folder name to reflect this,